cloudformation/cloudformation-template.json

```diff
@@ -0,0 +1,68 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Resources": {
+        "LambdaLogProcessor": {
+            "Type": "AWS::Lambda::Function",
+            "Properties": {
+                "Handler": "AWSLoggerLambda::AWSLoggerLambda.Function::FunctionHandler",
+                "Runtime": "dotnetcore3.1",
+                "Code": {
+                    "S3Bucket": "loggerstorage",
+                    "S3Key": "aws-task-lambda.zip"
+                },
+                "Description": "",
+                "MemorySize": 256,
+                "Timeout": 15,
+                "Role": "arn:aws:iam::375297447026:role/service-role/LogProcessorLambda-role-cw3e6ay0",
```

Comment:
You should create IAM roles through `CloudFormation` templates and reference them